cherryontech/acesquad
# Jiachun/nm 2/nm 51/fix radio buttons

Link to storycard:   https://acesquad.atlassian.net/browse/NM-51?atlOrigin=eyJpIjoiMmI3MTIzYWUwYzJmNGM5OGJjYjZlMTk1YzU4OGQxZmQiLCJwIjoiaiJ9


Description of work done:  
- add aria-label for textarea and radio buttons
- change form's heading title from h3 to h1 according to accessibility heading level rule
- add styling after changing heading title to make it still suit the form properly


Testing instructions:  
- Click "Create an offer or request" to view the form, and you can see the difference with accessibility assistance


What I learned:  
- The importance of having accessibility



Co-authored-by: Stephanie Page <vividvoltage@users.noreply.github.com> 
Co-authored-by: Lee O'Connor <lee-oconnor@users.noreply.github.com>
Co-authored-by: Jiachun Xiang <mpa-mxiang@users.noreply.github.com> 
Co-authored-by: Young He <youngrighthere@users.noreply.github.com> 
Co-authored-by: Delia <Parseluni@users.noreply.github.com> 
Co-authored-by: Carlisha Wilson <carlishawilson@users.noreply.github.com>
Co-authored-by: Ezgi Sari <sarielifezgi@users.noreply.github.com>
